cypress/integration/Button.spec.ts

```diff
@@ -31,7 +31,7 @@ describe('Button', () => {
     });
 
     it('should render the correct text', () => {
-      cy.get('button')
+      cy.findAllByRole('button', {name: 'Click Me'})
```

Comment:
Why are we using `findAllByRole` here when this story only renders 1 button? Could we use `findByRole` instead?

Reply:
Thank for catch! This replacement was copied from previous one, it should be changed.